server/policy.go

```diff
@@ -13,7 +19,69 @@ import (
 // will describe in what way the state failed. See the Policy documentation for
 // more information about the specifics of different policies.
 func EvaluatePolicy(state *pb.MachineState, policy *pb.Policy) error {
```

Comment:
Me want rego policy

Not related to your change, but would you be open to this being completely moved to a rego policy in the near future?

Reply:
I would certainly. The issue is primarily populating the Rego context to know that the signed values are even available and their verification status.